src/frontend/apps/desk/src/features/teams/assets/icon-open-close.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
By removing the width and the height here, you will be able to resize from the component the size of the icon.
```suggestion
<svg viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Same for the other icon.